daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/Action.scala

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.lf
+
+import com.daml.lf.data.Ref._
+import com.daml.lf.value.Value
+
+sealed abstract class Action extends Product with Serializable {
+  val templateId: TypeConName
+}
+
+object Action {
```

Comment:
I find this a bit confusing. To me, a command is something I’m requesting to happen vs an action is something that has happened and is recorded in a transaction. That might sound like a small distinction but it is an important one. E.g., an action always has informees. A command might result in something that crashes while evaluating the signatories or for another reason and does not have informees.

So I’d probably keep the commands name.

Reply:
I am more than happy to discuss this point. 

I took the name `Action` from canton `ActionDescription` whish include in addition of my `Action` the node seed (for create and exercise) and the LF version. I could rename my `Action` to `ActionDescription` and include the seed. Note sure if we really need the version wish is used only to rebuild the Node. 

I would be happy to get opinion of @andreaslochbihler-da.

Reply:
Rather than looking at Canton’s implementation let’s look at the ledger model: In there an action always has informees. This is not the case for a command, it can fail to produce a list of signatories, stakeholders or for a numerous other reasons.

I’d prefer to stick to `Command` as being something that we feed into speedy to produce a transaction whereas an action is a Node in a (partial) transaction which therefore always has informees and other stuff.

Reply:
OK. I reverted the last commit.